pavelib/assets.py

```diff
@@ -776,24 +775,12 @@ def webpack(options):
     """
     Run a Webpack build.
     """
-    def json_format_setting(setting_value):
-        """
-        Replaces capitalized booleans with booleans that
-        are compatible with parsed JSON in javascript.
-        """
-        if isinstance(setting_value, str):
-            # replace python bools with json valid bools
-            setting_value = re.sub(r'(\:\s?)True', r'\1true', setting_value)
-            setting_value = re.sub(r'(\:\s?)False', r'\1false', setting_value)
-
-        return setting_value
-
     settings = getattr(options, 'settings', Env.DEVSTACK_SETTINGS)
     result = Env.get_django_settings(['STATIC_ROOT', 'WEBPACK_CONFIG_PATH'], "lms", settings=settings)
     static_root_lms, config_path = result
     static_root_cms, = Env.get_django_settings(["STATIC_ROOT"], "cms", settings=settings)
-    js_env_extra_config_setting, = Env.get_django_settings(["JS_ENV_EXTRA_CONFIG"], "cms", settings=settings)
-    js_env_extra_config = json.dumps(json_format_setting(js_env_extra_config_setting) or {})
+    js_env_extra_config_setting, = Env.get_django_json_settings(["JS_ENV_EXTRA_CONFIG"], "cms", settings=settings)
+    js_env_extra_config = json.dumps(js_env_extra_config_setting or {})
```

Comment:
I think that here `js_env_extra_config_setting` is a string right? So `"string" or {}` seems like inconsistent typing - should it be `js_env_extra_config_setting or "{}"` ? Or is `js_env_extra_config_setting` an object already?

Reply:
`js_env_extra_config_setting` is a string, yes.

You're right, it is inconsistent. I'll update this to `js_env_extra_config_setting or "{}"`. In addition, I'll update the tests too, I added comments to address those changes 👍🏼

Reply:
Addressed 👍🏼